docs/application/web/api/6.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm

```diff
@@ -16,14 +16,6 @@ <h1>Checkbox and Radio Button</h1>
 <h2>Checkbox</h1>
 <img src="../../images/checkbox-wearable.png" alt="Photo of Checkbox in Tizen">
 <br>
-   <button
-      onclick="
-         if (navigator.userAgent.toLowerCase().indexOf('chrome') == -1) {
-            alert('Chrome Browser required')
-         } else
-            window.open('https://code.tizen.org/demos?path=1.1%2Fexamples%2Fwearable%2FUIComponents%2Fcontents%2Fcontrols%2Fcheckbox.html', '_blank')"
-   >Use example</button>
-   </button>
 <br>
 <h2>Radio Button</h2>
 <br>
```

Comment:
Question. While checking https://github.com/Samsung/tizen-docs/pull/1671, I found the button in line 32 is not removed.
Same issue to 6.0
Is it OK to keep it?